### PR TITLE
Milestone 1: Setting up application 

### PR DESCRIPTION
**Pull request by 8Bts**
In this branch:
- all required dependencies are installed
- unnecessary files deleted
- application deployed to Heroku

### CODE REVIEW

**Review by mahmoud717 — CHANGES_REQUESTED** [review]
### Changes required ♻️ 

**Hello @8Bts**

Great job with the project so far it looks amazing, but there are still some changes to be made:
 
---
### Requested 🔴 

- [ ] Please make sure to fix your linters errors, try deleting the package-lock.json file before pushing. generally, it's a good idea to delete it before pushing or deploying it.


---

please make these changes and request another code review.

if you have any questions don't hesitate to contact me on slack **MahmoudMohammad**

File: README.md [hunk 1]
```diff
@@ -1 +1,54 @@
...
+
+## Built With
+
+- HTML,
+- CSS,
+- ES6,
+- React.js
+- Redux
+
+## Live Demo
```
Comment (bellom) [inline README.md:14]: You should include your Heroku link here.

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @8Bts  :clap: 

### Awesome job adding `react-redux`, `redux`, and `prop-types`. 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Changes Required! ♻️ 

**1. Kindly check out my comments below;**

**2. For some reason, I could not start your app;**
I think you need to take a look at No. `8` and `9`
image


### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Review by Forison — APPROVED** [review]
## Status::Approved✔️

# 👋 @8Bts 

#### Good job setting up your calculator project as required. Unwanted boilerplates files have been removed and the required packages have been added. Kindly go ahead and merge and proceed to the next milestone.🤗🤗🤗 

#### Happy coding😎

Addo Forison

**Pull request merged**